### ISSUE TITLE
Disable SymPy Live from docs.sympy.org

### ISSUE THREAD

**Issue by bertiewooster** [sympy/sympy#22835]
I'd like to disable SymPy Live from the docs generally. The one bit I want to click on is the code to copy/paste things and it brings up SymPy Live instead. The SymPy Live box usually doesn't even work because it hasn't executed the previous examples from the same docstring.

_Originally posted by @oscarbenjamin in https://github.com/sympy/sympy/issues/22716#issuecomment-1009532721_

**Comment by faze-geek** [comment]
> The one bit I want to click on is the code to copy/paste things and it brings up SymPy Live instead. The SymPy Live box usually doesn't even work because it hasn't executed the previous examples from the same docstring.

Yes this is correct ,unknowing of such raised issues, I have dealt with this and wanted to report it too ! I belive sympy Live should be disabled .
Thanks @bertiewooster .
